### DISCUSSION TITLE
Open MS Teams meetings outside of Ferdium or popout window

### DISCUSSION THREAD

**Discussion by anthonyg2003** [ferdium/ferdium-app#1725] [Q&A] [answered]
I'm using Ferdium to house various chat platforms including MS Teams. Problem comes up when I need to join a Teams meeting. The meetings open within Ferdium and take up the whole Ferdium window, and so then in order to access other parts of the MS Teams (chats, channels, apps, etc.) during the meeting, the only choice is to minimize the meeting, which then puts the meeting in a little window that can't be resized.  Screenshots below.  I have the "Open URLs within Ferdium" slider to OFF.

Is there any way to either force URL links (like the meeting link in an MS Teams meeting) to open in an external browser, short of copy-pasting the meeting URL into browser?

Or even better, any way to open a meeting as a separate pop-out window so that I can have the meeting go-on while I continue to work in Ferdium?

![Screenshot 2024-04-28 122806](https://github.com/ferdium/ferdium-app/assets/19936471/f01cdfef-8839-457f-9d70-c7922a6a1083)

![Wioaka6ERS](https://github.com/ferdium/ferdium-app/assets/19936471/6f65564c-2228-42be-8bc1-1732ad2e2af9)

**Comment by vraravam** [comment]
@anthonyg2003 : The behavior you describe is how Teams behaves in any normal browser. Since Ferdium is only rendering each webpage in a chromium tab, there's nothing that we can control of this behavior.

**Comment by anthonyg2003** [comment, marked as answer]
> @anthonyg2003 : The behavior you describe is how Teams behaves in any normal browser. Since Ferdium is only rendering each webpage in a chromium tab, there's nothing that we can control of this behavior.

Ah, ok, that makes sense. And while the Teams desktop app opens meetings in a separate window, now after testing in browser, I see that Teams opens meetings in same window, same as seen in Ferdium. And when in browser, clicking on meeting link in Teams calendar also gives no option to open in a different browser window, so the behavior in Ferdium is as expected.

**For others looking for a workaround, this is what I came up with:** 

Option 1: Copy the URL from the Teams meeting and paste into your regular browser. Would of course involve logging into microsoft in that browser.

Option 2: Add an additional instance of Teams in Ferdium, and use one for meetings and the other for chat/apps. 

Option 3: Add "Custom Website" in Ferdium, and login to your microsoft account there. Then you can paste meeting links into that browser.

Of these, I'll probably use Option 1 so that I can keep meeting windows open on a different screen while leaving ferdium available for chats in a smaller window, and also so that I don't have to flip between tabs in Ferdium to get back to the meeting.
